e2e/etcd_test.go

```diff
@@ -451,14 +472,17 @@ func (ep *etcdProcess) Stop() error {
 
 func (ep *etcdProcess) waitReady() error {
 	defer close(ep.donec)
-	return waitReadyExpectProc(ep.proc, ep.cfg.isProxy)
+	return waitReadyExpectProc(ep.proc, ep.cfg.isProxy, ep.cfg.isV2Migrate)
 }
 
-func waitReadyExpectProc(exproc *expect.ExpectProcess, isProxy bool) error {
+func waitReadyExpectProc(exproc *expect.ExpectProcess, isProxy, isV2 bool) error {
```

Comment:
waitReadExpecProc(exproc ..., readString string)

we should avoid bool arg when possible. we should try VERY HARD to not use two bools.